embrace-android-api/src/main/kotlin/io/embrace/android/embracesdk/spans/Workflow.kt

```diff
@@ -0,0 +1,7 @@
+package io.embrace.android.embracesdk.spans
+
+public interface Workflow {
+    public fun start(segmentName: String? = null): Boolean
+    public fun startSegment(segmentName: String): Boolean
```

Comment:
Any potential for confusion in how this is named, given `start` also has `segmentName` included as a parameter?